### ISSUE TITLE
test ordering causes fail 

### ISSUE THREAD

**Issue by philipnbbc** [bbc/nmos-mdns-bridge#7]
Found that the tests would fail on some occasions with errors such as (the error from tox is not helpful):
```
======================================================================
ERROR: test_base_resource (tests.test_mdnsbridge.TestmDNSBridgeAPI)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/test_mdnsbridge.py", line 76, in test_base_resource
    self.assert_method_has_path_and_returns('base_resource', "/{}/{}/{}/".format(APINAMESPACE, APINAME, APIVERSION), {"resources": [value + "/" for value in VALID_TYPES]})
  File "tests/test_mdnsbridge.py", line 52, in assert_method_has_path_and_returns
    self.assertEqual(getattr(self.UUT, method).mock_path, path, msg="method {} was inserted into the webapi with the incorrect path: {}, when expected {}".format(method, getattr(self.UUT, method).mock_path, path))
AttributeError: 'function' object has no attribute 'mock_path'
```

It turns out that the order in which the tests are executed is important. `python -m unittest -v tests.test_mdnsbridge tests.test_mdnsbridgeservice` succeeds but `python -m unittest -v tests.test_mdnsbridgeservice tests.test_mdnsbridge` fails.

This suggests that one test is mocking the other test.

**Comment by samdbmg** [comment]
See also #8

**Comment by simonrankine** [comment]
Fixed by #8

**Comment by simonrankine** [comment]
Fixed in #8
